src/pages/Schedule/index.jsx

```diff
@@ -0,0 +1,502 @@
+import { useEffect, useState } from 'react';
+import { Helmet } from "react-helmet";
+import { useLocalStorage } from "usehooks-ts";
+import Typography from "@mui/material/Typography";
+import Button from "@mui/material/Button";
+import Fab from '@mui/material/Fab';
+import AddIcon from '@mui/icons-material/Add';
+import DeleteIcon from "@mui/icons-material/Delete";
+import LockIcon from "@mui/icons-material/Lock";
+import LockOpenIcon from "@mui/icons-material/LockOpen";
+import Paper from '@mui/material/Paper';
+import TableContainer from '@mui/material/TableContainer';
+import TableHead from '@mui/material/TableHead';
+import TableRow from '@mui/material/TableRow';
+import TableCell from '@mui/material/TableCell';
+import TableBody from '@mui/material/TableBody';
+import Table from '@mui/material/Table';
+import TextField from '@mui/material/TextField';
+import Select from '@mui/material/Select';
+import MenuItem from '@mui/material/MenuItem';
+import Checkbox from '@mui/material/Checkbox';
+import Tooltip from '@mui/material/Tooltip';
+import InfoIcon from '@mui/icons-material/Info';
+import Alert from '@mui/material/Alert';
+import Snackbar from '@mui/material/Snackbar';
+import CircularProgress from '@mui/material/CircularProgress';
+import { useUserData, useSchedule } from '../../lib/teuthologyAPI';
+import { useMutation } from "@tanstack/react-query";
+import Editor from "react-simple-code-editor";
+import { highlight, languages } from "prismjs/components/prism-core";
+import Accordion from "@mui/material/Accordion";
+import AccordionSummary from "@mui/material/AccordionSummary";
+import AccordionDetails from "@mui/material/AccordionDetails";
+import ExpandMoreIcon from "@mui/icons-material/ExpandMore";
+export default function Schedule() {
+  const keyOptions =
+    [
+      "--ceph",
+      "--ceph-repo",
+      "--suite-repo",
+      "--suite-branch",
+      "--suite",
+      "--subset",
+      "--sha1",
+      "--email",
+      "--machine-type",
+      "--filter",
+      "--filter-out",
+      "--filter-all",
+      "--kernal",
+      "--flavor",
+      "--distro",
+      "--distro-version",
+      "--newest",
+      "--num",
+      "--limit",
+      "--priority",
```

Comment:
When I schedule runs, they are all end with "dead" status because for docker setup the `"<config_yaml>": ["/teuthology/containerized_node.yaml"]` config is needed. I hard-coded that in request and it resulted with "pass" status! 
Right now, t-api input of `<config_yaml>` is a path (like the above example) but there's a PR open which will take in actual config text: https://github.com/ceph/teuthology-api/pull/50. Nevertheless, I guess the react coded would probably be the same for both.

So we probably need to add support for `config_yaml` to get green runs in docker.  It's a `list` type though which sounds complex so maybe we can implement this in next PR.

Reply:
I never got to that part where I was looking for a passed run, so thank you for reaching that.